### ISSUE TITLE
[Bug Report] [Component] [tree-select] A wrong import in `tree-select` which can cause compile(build) error.

### ISSUE THREAD

**Issue by 7086cmd** [element-plus/element-plus#7008]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.2.31`
- Element Plus Version: `2.1.8`
- Browser / OS: `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.60 Safari/537.36 Edg/100.0.1185.29`
- Build Tool: `Vite`

### Related Component

- `el-tree-select`

### Reproduction Link

[Github Repo](https://github.com/7086cmd/element-plus-bug-tree-select.git)

### Steps to reproduce

1. Use `vite`
2. Install `unplugin-auto-import` and `unplugin-vue-components`
3. Configure `vite.config.ts` so that you can use 2 plugins normally.
4. Add `el-tree-select` into `App.vue`
5. Start vite dev server.
6. You will see that building error.

### What is Expected?

Could import `tree-select` successfully without handle it by ourself.

### What is actually happening?

It can not import this component this or we will recieve an error:
```shell
X [ERROR] Could not resolve "element-plus/theme-chalk/tree-select.css"

    node_modules/.pnpm/element-plus@2.1.8_vue@3.2.31/node_modules/element-plus/es/components/tree-select/style/css.mjs:3:7:
      3 │ import 'element-plus/theme-chalk/tree-select.css';
        ╵        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

  The module "./theme-chalk/tree-select.css" was not found on the file system:

    node_modules/.pnpm/element-plus@2.1.8_vue@3.2.31/node_modules/element-plus/package.json:30:11:
      30 │     "./*": "./*"
         ╵            ~~~~~

  You can mark the path "element-plus/theme-chalk/tree-select.css" as external to exclude it from
  the bundle, which will remove this error.

13:14:37 [vite] error while updating dependencies:
Error: Build failed with 1 error:
node_modules/.pnpm/element-plus@2.1.8_vue@3.2.31/node_modules/element-plus/es/components/tree-select/style/css.mjs:3:7: ERROR: Could not resolve "element-plus/theme-chalk/tree-select.css"     
    at failureErrorWithLog (E:\stunning-new\node_modules\.pnpm\esbuild@0.14.21\node_modules\esbuild\lib\main.js:1601:15)
    at E:\stunning-new\node_modules\.pnpm\esbuild@0.14.21\node_modules\esbuild\lib\main.js:1247:28
    at runOnEndCallbacks (E:\stunning-new\node_modules\.pnpm\esbuild@0.14.21\node_modules\esbuild\lib\main.js:1032:63)
    at buildResponseToResult (E:\stunning-new\node_modules\.pnpm\esbuild@0.14.21\node_modules\esbuild\lib\main.js:1245:7)
    at E:\stunning-new\node_modules\.pnpm\esbuild@0.14.21\node_modules\esbuild\lib\main.js:1354:14
    at E:\stunning-new\node_modules\.pnpm\esbuild@0.14.21\node_modules\esbuild\lib\main.js:664:9
    at handleIncomingPacket (E:\stunning-new\node_modules\.pnpm\esbuild@0.14.21\node_modules\esbuild\lib\main.js:761:9)
    at Socket.readFromStdout (E:\stunning-new\node_modules\.pnpm\esbuild@0.14.21\node_modules\esbuild\lib\main.js:631:7)
    at Socket.emit (node:events:390:28)
    at addChunk (node:internal/streams/readable:324:12)

```

### Additional comments

It might a error with the bundler of `element plus`. It is possible that there's something wrong in the bundler config for `tree-select`.

---

Besides, the file imported of `node_modules/.pnpm/element-plus@2.1.8_vue@3.2.31/node_modules/element-plus/es/components/tree-select/style/css.mjs:3:7` is `element-plus/theme-chalk/src/tree-select/css`, but it really not exists. However, I found the current file which is located in `element-plus/theme-chalk/el-tree-select.css`

**Comment by 7086cmd** [comment]
One possible solution might be #7009 .

**Comment by chenxch** [comment]
I took the time to test it today and found that there is no problem with your feedback. You may still need to check it from your cli.

**Comment by RexSheng** [comment]
@YunYouJun the file imported of node_modules/.pnpm/element-plus@2.1.8_vue@3.2.31/node_modules/element-plus/es/components/tree-select/style/css.mjs:3:7 is element-plus/theme-chalk/src/tree-select/css, but it really not exists.此问题导致项目升级到2.1.8后一直运行不了

**Comment by chenxch** [comment]
vite.config.ts
```

    Components({
      dts: true,
    })
```

**Comment by toimc** [comment]
Same here.

Env:

- node: 14LTS
- Element plus 2.1.8
- Vite: 2.9.1

Project: https://github.com/toimc-team/vue3-toimc-admin

**Comment by chenxch** [comment]
Sorry, this project could not be started at all.

**Comment by chenxch** [comment]
> vite.config.ts
> 
> ```
> 
>     Components({
>       dts: true,
>     })
> ```

@RexSheng @toimc Is this temporary solution not good, I tested it with his demo.

**Comment by chenxch** [comment]
![image](https://user-images.githubusercontent.com/23251408/162210009-1b5a48d6-3bad-466d-a846-f17fe5856191.png)
Of course, it has been tested. Well, let's think about how to fix this question. Your pr still has a writing problem, and my review part is also wrong.

**Comment by chenxch** [comment]
> Well, what is the repo of your example? Could you please provide one?

It's your project

**Comment by 7086cmd** [comment]
Oh.. :mask: But ... actually the main resolver is for `unplugin-vue-components`, and `components.d.ts`(the auto-generated import map for `unplugin-vue-components`) is full of types, while `auto-imports.d.ts`(the auto-generated import map for `unplugin-auto-import`) is empty.

Here are my pics in my project(*It faced the same issue and I edited the `css.mjs` in order to solve this problem*):
![image](https://user-images.githubusercontent.com/54303040/162214778-e519fbde-aac1-4903-a5d9-e442a00fc235.png)
![image](https://user-images.githubusercontent.com/54303040/162214804-9dc0c2a8-6cb3-4341-893e-95a5ab5b9d26.png)

**Comment by 7086cmd** [comment]
Well, look for the provider of `ElementPlusResolver`. It is `unplugin-vue-components`, isn't it?

**Comment by chenxch** [comment]
> Well, look for the provider of `ElementPlusResolver`. It is `unplugin-vue-components`, isn't it?

Yes, I am also using this component, this problem should be solved from `unplugin-vue-components`.

**Comment by chenxch** [comment]
> unplugin-vue-components

But I am also wondering if there is a mistake in one of the links that breaks the original rules.

**Comment by chenxch** [comment]
> unplugin-vue-components

`unplugin-vue-components` everything works fine, he found `/element-plus/es/components/tree-select/style/css.mjs` correctly

**Comment by chenxch** [comment]
@7086cmd I have found the correct solution and have given feedback in your pr, you can take a look. The problem is that the paths referenced by the `css.ts` and  `index.ts` inside the style are mixed up. `css.ts` should point to the `compiled file`. Thanks to this question I understand `element-plus` better.

**Comment by 7086cmd** [comment]
Well, @chenxch I viewed the other `css.ts` in different components, and it always refer to the compiled file... So I think #7009 is one of the possible solutions..., and I really hope the `el-tree-select` will be a useful and common component because it is really useful.

**Comment by chenxch** [comment]
> Well, @chenxch I viewed the other `css.ts` in different components, and it always refer to the compiled file... So I think #7009 is one of the possible solutions..., and I really hope the `el-tree-select` will be a useful and common component because it is really useful.

That's right, so check your workflows on your PR #7009  as soon as possible so they can check and merge.

**Comment by 7086cmd** [comment]
@chenxch The title of the pr (#7009) was incorrect, so I edited it while it did not re-run the workflow of pr check again. The auto test would run when the reviewer check it (?

**Comment by 7086cmd** [comment]
The workflow shows that:
> This workflow is awaiting approval from a maintainer in https://github.com/element-plus/element-plus/pull/7009

**Comment by chenxch** [comment]
Your `Semantic Pull Request` is a failure, and the acquisition is a naming issue of `patch-1`. 
cc @JeremyWuuuuu

**Comment by 7086cmd** [comment]
Might solved via #7009. I will close this issue first.
